src/components/MessagesList/MessagesList.vue

```diff
@@ -34,20 +34,24 @@ get the messagesList array and loop through the list to generate the messages.
 		@scroll="debounceHandleScroll">
 		<div v-if="displayMessagesLoader" class="scroller__loading icon-loading" />
 
-		<template v-for="item of messagesGroupedByAuthor">
-			<div v-if="item.dateSeparator" :key="`date_${item.id}`" class="messages-group__date">
-				<span class="messages-group__date-text" role="heading" aria-level="3">
-					{{ item.dateSeparator }}
-				</span>
+		<template v-for="[item, list] of Object.entries(messagesGroupedByDateByAuthor)">
```

Comment:
Would be perfect to go on with semantic list elements: `<ul />`, `<li />` here and in the MessagesGroup components
```suggestion
		<li v-for="[item, list] of Object.entries(messagesGroupedByDateByAuthor)">
```